ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bgp.schema.yml

```diff
@@ -2243,6 +2243,26 @@ keys:
                     transit:
                       type: bool
                       description: Enable EVPN multicast transit mode.
+            evpn_multicast_gateway_dr_election_algorithm:
+              type: dict
+              keys:
+                election_parameter:
+                  type: str
+                  description: |-
+                    DR election parameters:
+                      hrw: Default selection based on highest random weight.
+                      modulus: Selection based on VLAN ID modulo number of candidates.
+                      preference: Selection based on a configured preference value.
+                  valid_values:
+                    - hrw
```

Comment:
I do not see `hrw` on running-config tho option is available in cli.
should we keep it ?

Reply:
it is there if you do "show active all" to include default config.